jax/_src/scipy/stats/kde.py

```
@@ -0,0 +1,249 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from functools import partial
+from typing import Any, NamedTuple
+
+import numpy as np
+import scipy.stats as osp_stats
+
+import jax.numpy as jnp
+from jax import jit, random, vmap
+from jax._src.numpy.lax_numpy import _promote_dtypes_inexact
+from jax._src.numpy.util import _wraps
+from jax.scipy import linalg, special
+
+
+@_wraps(osp_stats.gaussian_kde, update_doc=False)
+def gaussian_kde(dataset, bw_method=None, weights=None):
+  dataset = jnp.atleast_2d(dataset)
```

Should we raise an error if the dataset is complex?